build.sbt

```diff
@@ -5,7 +5,7 @@ logLevel := Level.Debug
 
 // this values should be in sync with ergo-wallet/build.sbt
 val scala211 = "2.11.12"
-val scala212 = "2.12.18"
+val scala212 = "2.12.11"
```

Comment:
prior to 2.12.18 it was 2.12.10, possibly revert to this.

Changing commit: https://github.com/ergoplatform/ergo/commit/7865dd4ce99ece9c06b8beebe555b05086e1af4a#diff-57d3c344906eae6433d7decac6004ce19dcd27a73c6cda7f59ef95939ffe63b7

Reply:
> thing is addressed already

Could you send a PR with the relevant changes to avoid the reversion?

Or at least clarify what the concrete fix is?

Reply:
> updating `scala-async` does the trick

Great!

Reply:
@ccellado , tried scala-async update + `-Xasync`, but can't get it to work.

Fetched your `docker-fix` branch: a similar error (within IntelliJ)

```sh
P:\work\ergo\src\it\scala\org\ergoplatform\it\DeepRollBackSpec.scala:52:44
The async requires the compiler option -Xasync (supported only by Scala 2.12.12+ / 2.13.3+)
    val result: Future[Unit] = Async.async {
```

(cli complile via `sbt it:compile it2:compile` works though)